utils/config/daos_server.yml

```diff
@@ -177,13 +177,13 @@
 ## Number of hugepages to allocate for DMA buffer memory
 #
 ## Specifies the number (not size) of hugepages to allocate for use by NVMe
-## through SPDK. Note that each target requires 1 GB of hugepage space, so
+## through SPDK. Note that each target requires 1 GiB of hugepage space, so
 ## this value needs to represent the total amount of hugespace required for
 ## all targets across all engines on host, divided by the system hugepage size.
-## If not set here, an appropriate value will be automatically calculated and
-## default system hugepage size will be used.
+## If not set here, an appropriate value will be automatically calculated based
+## on the number of targets (using the default system hugepage size of 2MiB).
```

Comment:
this change isn't quite correct, as mentioned in the original text hugepages of the size specified in the system will be used, it will not force the use of 2MiB pages

Reply:
"using the default system hugepage size of 2MiB" -> "using the system configured hugepage size"

Reply:
ok, fixed.